Scripts/RemediationScripts/Remediate-EnableSSLDBForMySQLFlexibleServer.ps1

```diff
@@ -0,0 +1,829 @@
+                }
+            }
+            $log | ConvertTo-json -depth 10  | Out-File $logFile
+        }
+    }
+    else {
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "[Step 4 of 4] Enable SSL on Azure Database for MySQL flexible server(s) in the Subscription." 
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "Skipped as -DryRun switch is provided." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "Next steps:" -ForegroundColor $([Constants]::MessageType.Info)
+        Write-Host "Run the same command with -FilePath [$($backupFile)] and without -DryRun, to Enable SSL for Azure Database for MySQL flexible server(s) listed in the file."
+        Write-Host $([Constants]::DoubleDashLine)
+    }
+}
+
+function Disable-SSLForDBForMySQLFlexibleServer {
+    <#
+        .SYNOPSIS
+        Rolls back remediation done for 'Azure_DBForMySQLFlexibleServer_DP_Enable_SSL' Control.
+
+        .DESCRIPTION
+        Rolls back remediation done for 'Azure_DBForMySQLFlexibleServer_DP_Enable_SSL' Control.
+        Change SSL to Previous Value on Azure Database for MySQL flexible server(s) in the Subscription. 
+        
+        .PARAMETER SubscriptionId
+        Specifies the ID of the Subscription that was previously remediated.
+        
+        .PARAMETER Force
+        Specifies a forceful roll back without any prompts.
+        
+        .Parameter PerformPreReqCheck
+        Specifies validation of prerequisites for the command.
+      
+        .PARAMETER FilePath
+        Specifies the path to the file to be used as input for the roll back.
+
+        .INPUTS
+        None. You cannot pipe objects to Disable-SSLForDBForMySQLFlexibleServer.
+
+        .OUTPUTS
+        None. Disable-SSLForDBForMySQLFlexibleServer does not return anything that can be piped and used as an input to another command.
+
+        .EXAMPLE
+        PS> Disable-SSLForDBForMySQLFlexibleServer -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\SetSSLForDBForMySQLFlexibleServer\RemediatedDBForMySQLFS.csv
+
+        .LINK
+        None
+    #>
+
+    param (
+        [String]
+        [Parameter(Mandatory = $true, HelpMessage = "Specifies the ID of the Subscription that was previously remediated.")]
+        $SubscriptionId,
+
+        [Switch]
+        [Parameter(HelpMessage = "Specifies a forceful roll back without any prompts")]
+        $Force,
+
+        [Switch]
+        [Parameter(HelpMessage = "Specifies validation of prerequisites for the command")]
+        $PerformPreReqCheck,
+
+        [String]
+        [Parameter(Mandatory = $true, HelpMessage = "Specifies the path to the file to be used as input for the roll back")]
+        $FilePath
+    )
+
+    Write-Host "[Step 1 of 3] Validate and install the modules required to run the script."
```

Comment:
Kindly do changes for this step in similar way as suggested for remediation function. Simple copy and paste will work as well as they are same step,